src/interface/mesh_data.hpp

```diff
@@ -419,6 +412,7 @@ class MeshData {
     bvars_cache_.clear();
   }
 
+  int GetNDim() const { return ndim_; }
```

Comment:
This change is so that we can create a `MeshData` object without a `Mesh` and just an `ndim` and block list for regression tests, right? Nice!

Reply:
Yep that's right! :)